nats-base-client/headers.ts

```diff
@@ -182,24 +182,24 @@ export class MsgHdrsImpl implements MsgHdrs {
     return k.trim();
   }
 
-  get(k: string): string {
-    const key = MsgHdrsImpl.canonicalMIMEHeaderKey(k);
+  get(k: string, mime = true): string {
+    const key = mime ? MsgHdrsImpl.canonicalMIMEHeaderKey(k) : k;
```

Comment:
The check would work better being case insensitive, during the transport of the headers any library may transform the header etc.. but retrieving a key "foo", should be the same as getting "Foo" for example, but then also may want an option to get the exact match.